cms.env.json

```diff
@@ -0,0 +1,160 @@
+                                                                                                                                                    {
```

Comment:
@clrux seems adding  Studio configuration .json file here is not the right way of updating it. Please ask someone from devops, may be @e0d .